src/zap/zapheaders.cpp

```diff
@@ -292,13 +293,18 @@ void ZapDebugDirectory::SaveNGenDebugDirectoryEntry(ZapWriter *pZapWriter)
     debugDirectory.Type = IMAGE_DEBUG_TYPE_CODEVIEW;
     debugDirectory.SizeOfData = m_pNGenPdbDebugData->GetSize();
     debugDirectory.AddressOfRawData = m_pNGenPdbDebugData->GetRVA();
+    // NGen PDBs are always Windows currently so make sure the "is portable pdb"
```

Comment:
Also what is the "is portable pdb" indicator? (Is it the MinorVersion?, then just say that)